Jenkinsfile_CNP

```diff
@@ -42,6 +42,7 @@ withPipeline(type, product, component) {
     builder.gradle('integration')
   }
 
+  enableDbMigration(dbMigrationVaultName = 'reform-scan')
```

Comment:
it should be `enableDbMigration('reform-scan')` or you sure current one works ?

Reply:
?